src/user/user-credentials/CredentialsResetActionMultiSelect.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Controller, useFormContext } from "react-hook-form";
+import {
+  FormGroup,
+  Select,
+  SelectOption,
+  SelectVariant,
+} from "@patternfly/react-core";
+
+import { RequiredActionAlias } from "@keycloak/keycloak-admin-client/lib/defs/requiredActionProviderRepresentation";
+import { HelpItem } from "../../components/help-enabler/HelpItem";
+
+export const CredentialsResetActionMultiSelect = () => {
+  const { t } = useTranslation("users");
+  const { control } = useFormContext();
+  const [open, setOpen] = useState(false);
+
+  return (
+    <FormGroup
+      label={t("resetActions")}
+      labelIcon={
+        <HelpItem
+          helpText="clients-help:resetActions"
+          fieldLabelId="resetActions"
+        />
+      }
+      fieldId="actions"
+    >
+      <Controller
+        name="actions"
+        defaultValue={[]}
+        control={control}
+        render={({ onChange, value }) => (
+          <Select
+            toggleId="actions"
+            variant={SelectVariant.typeaheadMulti}
+            chipGroupProps={{
+              numChips: 3,
+            }}
+            menuAppendTo="parent"
+            onToggle={(open) => setOpen(open)}
+            isOpen={open}
+            selections={value.map((o: string) => o)}
```

Comment:
Why map this value to itself? Can it not be passed directly?
```suggestion
            selections={value}
```